.github/workflows/ci.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [macos-latest]
-        node-version: [16.x, 18.x, 20.x, 22.x]
+        node-version: [18.x, 20.x, 22.x]
         include:
```

Comment:
Let's make the python version part of the matrix and add an additional include entry for node 16.x + python 3.10.